src/audit_emergency_merges.ts

```diff
@@ -0,0 +1,14 @@
+import { getMergedEmergencyPRsMissingReview } from './github';
+import { postMessage } from './slack';
+
+export async function auditEmergencyMerges() {
+  const pullRequests = await getMergedEmergencyPRsMissingReview();
+
+  for(const pr of pullRequests) {
```

Comment:
```suggestion
  for (const pr of pullRequests) {
```
Surprised out linter didn't pick this up.